hw/dv/tools/dvsim/dsim.hjson

```diff
@@ -14,7 +14,9 @@
                "+incdir+{UVM_HOME}/src",
                "{UVM_HOME}/src/uvm_pkg.sv",
                // Add dpi/vpi include path.
-               "-c-opts -I{DSIM_HOME}/include",
+               "-c-opts -I{DSIM_HOME}/../../include",
```

Comment:
It is counter-intuitive to include a directory that is not within `DSIM_HOME`. This points to a setup that is specific to us. We need this to work for other companies and individuals who have licensed DSIM as well.

Reply:
Our dsim installation is quirky, so we may have add a DSIM_INCLUDE variable in the eda/workspace_setup.sh file. The quirkiness it that we set DSIM_HOME to in dsim/bin/latest instead of plain dsim.